### ISSUE TITLE
Could this code be used for 3090ti

### ISSUE THREAD

**Issue by wangmiaowei** [mosaicml/diffusion#28]
I am limited GPU resources and the maximum GPU is 3090ti. Could I use your code to finetune stable diffusion under 3090ti?

**Comment by Landanjs** [comment]
Copying message from community slack: We haven’t done any testing on 3090ti, but this will depend on the batch size, gradient accumulation (microbatchsize), and resolution. At a batch size of 64 and resolution 256x256, we are seeing ~30G of memory. So a batch size of 16 or maybe 32 should fit on 24G of memory, but again, we’ve only done tests on A100s.
